src/monopoly.js

```diff
@@ -64,39 +64,45 @@ export default class Monopoly {
       this.toggleListenersOff();
 
       const diceArray = Monopoly.rollDice();
-      console.log('dice: ' + diceArray.reduce(( acc, cur ) => acc + cur, 0));//test
       activePlayer.movePlayer(diceArray, this.propertyArray.length);
+
+      console.log('dice: ' + diceArray.reduce(( acc, cur ) => acc + cur, 0));//test
       activePlayer.money -= diceArray.reduce(( acc, cur ) => acc + cur, 0) * 10;//test
-      console.log('new position ' + activePlayer.position + '. funds left: ' + activePlayer.money);//test
-//      this.landOnFunction(activePlayer);TODO
-
-      if(diceArray.every(element => element == diceArray[0])){//probably too much.  allows for more than 2 dice
-        activePlayer.doubles++;
-        if(activePlayer.doubles >= 3){
-          activePlayer.doubles = 0;
-//          this.goToJail(activePlayer);
+      console.log('new position ' + this.propertyArray[activePlayer.position].nameStr + '. funds left: ' + activePlayer.money);//test
+
+      this.emitter.once('finishTurn', () => {
+        if(diceArray.every(element => element == diceArray[0])){//probably too much.  allows for more than 2 dice
+          activePlayer.doubles++;
+          if(activePlayer.doubles >= 3){
+            activePlayer.doubles = 0;
+  //          this.goToJail(activePlayer);
+          }
+          else {
+            this.runTurn(activePlayer);
+            return this;
+          }
         }
         else {
-          this.runTurn(activePlayer);
-          return this;
+        activePlayer.doubles = 0;
         }
-      }
-      else {
-      activePlayer.doubles = 0;
-      }
 
-      this.currentPlayer++;//or next index or w.e. I use
-      if(this.currentPlayer >= this.playerArray.length){
-        this.currentPlayer = 0;
-      }
+        this.currentPlayer++;//or next index or w.e. I use
+        if(this.currentPlayer >= this.playerArray.length){
+          this.currentPlayer = 0;
+        }
+
+        this.emitter.emit('nextPlayer', () => {});
+        return this;
+      });
+
+      this.propertyArray[activePlayer.position].landOnFunction(activePlayer, this.emitter);
 
-      this.emitter.emit('nextPlayer', () => {});
       return this;
     });
 
     this.toggleListenersOn();
     console.log('Toggling Listers\n promptRolls for ' + activePlayer.person);//test
-    this.emitter.emit('promptRoll', activePlayer, () => {});
+    this.emitter.emit(('promptRoll' + activePlayer.socketID) , activePlayer, () => {});
```

Comment:
I like string interpolation in es6 opposed to manually adding together strings:

```
`promptRoll${activePlayer.socketID}`
```

String interpolation occurs with backticks.